src/server/req_signal.c

```diff
@@ -112,19 +113,20 @@ req_signaljob(struct batch_request *preq)
 	if (parent == (job *)0)
 		return;		/* note, req_reject already called */
 
-	if ((strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_RESUME) == 0) ||
-		(strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_SUSPEND) == 0)) {
+	if (strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_RESUME) == 0 || strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_ADMIN_RESUME) == 0)
+		susrsm = -1;
```

Comment:
The use of susrsm is preexisting code.  I'll make it consistent.  It'll make req_signaljob() easier to read.